Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:16.04
+
+RUN apt-get -yqq update
+RUN apt-get -yqq install nodejs npm
+RUN apt-get -yqq install curl
+RUN apt-get -yqq install python3
+RUN ln -s /usr/bin/nodejs /usr/bin/node
+
+ADD tiles /opt/tile-demo
+WORKDIR /opt/tile-demo
+
+RUN node -v
+RUN npm cache clean -f
+RUN npm install -g n
+RUN n 6.11.3
+RUN npm install
+RUN node -v
+RUN npm run build
+
+
+EXPOSE 8000
+
+CMD ["python3", "-m" , "http.server"]
```

Comment:
You might also be able to run a dev server with nodejs, e.g. with [webpack-dev-server](https://webpack.js.org/configuration/dev-server/).

If node is the only thing you need in your container, you could probably use the [official node image](https://hub.docker.com/_/node/) instead of ubuntu.

Feel free to keep these things in mind for a later commit / PR, no need to change it now.